lib/ansible/modules/source_control/subversion.py

```diff
@@ -132,12 +140,19 @@ def _exec(self, args, check_rc=True):
             '--trust-server-cert',
             '--no-auth-cache',
         ]
+        stdin_data = None
         if self.username:
             bits.extend(["--username", self.username])
         if self.password:
-            bits.extend(["--password", self.password])
+            if self.has_option_password_from_stdin():
+                bits.extend(["--password-from-stdin"])
```

Comment:
Since this is just adding a single parameter, `append` can be used.
```suggestion
                bits.append("--password-from-stdin")
```